pkg/apis/serving/v1alpha1/openapi_generated.go

```diff
@@ -23,53 +23,53 @@ limitations under the License.
 package v1alpha1
 
 import (
-	openapispec "github.com/go-openapi/spec"
+	spec "github.com/go-openapi/spec"
```

Comment:
any reason to change this ? spec is the default alias name though.

Reply:
Same reason as above. @jinchihe will make specific changes needed for openapi-gen

Reply:
@johnugeorge If you updated the openapi_generated.go by running the script `hack/update-openapigen.sh`, should no changes, seems this is updated [here](https://github.com/kubeflow/kfserving/blob/fb4a31ddf1bce6a0c2c1b6c65068981bb93b309b/hack/update-openapigen.sh#L34). Personally you can leave the file, just upgrade knative, I can do this later. Or of you want to do that, I can work this with you together. Thanks.

Reply:
updated